agent/runtime-attach/src/main/java/com/microsoft/applicationinsights/attach/ApplicationInsights.java

```diff
@@ -69,6 +73,15 @@ private static Optional<String> findJsonConfig() {
     }
   }
 
+  private static String findJsonConfigFile() {
+    String fileFromProperty =
+        System.getProperty(APPLICATIONINSIGHTS_RUNTIMEATTACH_CONFIGURATION_FILE);
```

Comment:
or support both?

Reply:
The configuration of a spring profile with a system property (`-Dspring.profiles.active=dev`) may be more common than the configuration with an environment variable (`export spring_profiles_active=dev`).  We may start with a system property, and following user feedbacks, we could add an env var. The system property would allow the ability to configure applications (JVM) in different ways in the same environment, which would not be possible with an env var. I prefer to wait for user feedback and don't multiply the ways of configuring Application Insights.